### ISSUE TITLE
ah yes app market, here we go again

### ISSUE THREAD

**Issue by kingofsponges** [IgorTimofeev/MineOS#329]
remember the time the app market was down? **this issue is back.**
are the servers down on some purpose? or are those server issues?

**Comment by IgorTimofeev** [comment]
Yep, server was down. Why? Who knows... maybe moon was in bad phase
